networkx/algorithms/centrality/betweenness_subset.py

```diff
@@ -187,11 +187,13 @@ def edge_betweenness_centrality_subset(G,sources,targets,
     return b
 
 # obsolete name
-def betweenness_centrality_source(G,normalized=True,weight=None,sources=None):
+def betweenness_centrality_source(G, normalized=True, weight=None,
+                                  sources=None):
     if sources is None:
-        sources=G.nodes()
-    targets=G.nodes()
-    return betweenness_centrality_subset(G,sources,targets,normalized,weight)
+        sources = list(G)
```

Comment:
It looks like sources and targets could be left as iterators here.

Also, betweenness_subset doesn't show up on the documentation page. Perhaps that is another PR though.